### PR TITLE
Initial version of migration guide

### PR DESCRIPTION
**Pull request by berndruecker**

### CODE REVIEW

File: docs/guides/migrating-from-Camunda-Platform.md [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+Please note the following important considerations:
+
+1. **Migrating from Camunda Platform to Camunda Cloud is not a paved path**. As you will read below, there are several differences between the products and not all solutions are easy to migrate. This guide describes our experiences with customer projects so far, but is not a "turn-key"-migration procedure.
+
+2. **It is not necessarily recommended to migrate from Camunda Platform to Camunda Cloud**. Camunda Platform is a great product with existing support; there is no need to migrate existing solutions unless you have a good reason to (e.g. run it on SaaS, certain requirements on scalability or fault-tolerance, etc.) Therefore, we view migration more as an exception than a rule. Still, knowing the basic steps of migration can be helpful in building solutions with Camunda Platform, that will make migration easier in the future if necessary.
+
+Planning to migrate your project? [Talk to us](https://camunda.com/contact/) so we can help where needed.
```
Comment (akeller) [inline docs/guides/migrating-from-Camunda-Platform.md:14]: Do we want to route everyone through this Contact Us form? Or do we want to have options for the community forums and Slack as well? If so, we should link to this page - https://docs.camunda.io/contact

File: docs/guides/migrating-from-Camunda-Platform.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+* Discuss external task adapter for Java code and probably add it to the [Camunda Platform to Cloud Adapter](https://github.com/berndruecker/camunda-platform-to-cloud-migration/tree/main/camunda-platform-to-cloud-adapter).
+* Discuss more concepts around BPMN
+** [Field injection](https://docs.camunda.org/manual/latest/user-guide/process-engine/delegation-code/#field-injection) that is using ```camunda:field``` available on many BPMN elements.
+** Multiple instance markers available on most BPMN elements.
+** ```camunda:inputOutput``` available on most BPMN elements.
+** ```camunda:errorEventDefinition``` available on several BPMN elements.
+
+And even more. 
+
+Please [reach out to us](https://camunda.com/contact/) to discuss your specific migration use case.
```
Comment (akeller) [inline docs/guides/migrating-from-Camunda-Platform.md:387]: Same comment here, would this contact page make more sense - https://docs.camunda.io/contact ?

**Review by akeller — COMMENTED** [review]
Just a question about the contact details, otherwise this looks good!

**Comment by akeller** [comment]
@berndruecker thoughts on the contact page link?

File: docs/guides/migrating-from-Camunda-Platform.md (hunk 1 above)
Comment (berndruecker) [reply docs/guides/migrating-from-Camunda-Platform.md:14]: I think you are right - linking that page makes more sense. Thx! I will take care of updating the PR

Comment (berndruecker) [reply docs/guides/migrating-from-Camunda-Platform.md]: https://github.com/berndruecker/camunda-cloud-documentation/commit/fc4bc097ce039eb722cbf6f328277bdd08119658

File: docs/guides/migrating-from-Camunda-Platform.md (hunk 2 above)
Comment (berndruecker) [reply docs/guides/migrating-from-Camunda-Platform.md]: https://github.com/berndruecker/camunda-cloud-documentation/commit/fc4bc097ce039eb722cbf6f328277bdd08119658

**Review by akeller — APPROVED** [review]

**Pull request merged**